yeelight_light_fancl5.yaml

```diff
@@ -136,7 +226,64 @@ xiaomi_ylyk01yl:
             args: ['int(keycode)']
 
   on_long_press:
-    # Trigger on any keycode
+    # night light
+    - keycode: 1
+      then:
+       - light.toggle: night_light
+       - switch.turn_on: buzzer_beep
+       
+    # stepless cycle control speed
+    - keycode: 2
+      then:
+        - fan.cycle_speed: yeelight_ceiling_fan
+        - switch.turn_on: buzzer_beep
+       
+    # fan mode REVERSE select for circle winds
+    - keycode: 4
+      then:
+        - fan.turn_on:
+           id: yeelight_ceiling_fan
+           direction: reverse
+        - switch.turn_on: buzzer_beep
+
+    # color_temperature control，only support by ceiling light (154K-370K)
+    # color_temperature minus
+    - keycode: 4
```

Comment:
Here you change the color temperature also by **keycode 4**. This looks wrong to me.

Reply:
wait me a second, I test again